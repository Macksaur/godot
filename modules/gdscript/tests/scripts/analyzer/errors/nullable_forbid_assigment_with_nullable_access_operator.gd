class A:
    var b: String
func test():
    var a: A? = A.new()
    a?.b = "123"
    print(a?.b)
