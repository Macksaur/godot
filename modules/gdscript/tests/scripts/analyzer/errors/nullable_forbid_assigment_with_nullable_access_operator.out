GDTEST_OK
123
